core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -2758,7 +2759,7 @@ public InitMilestone getInitLevel() {
         return initLevel;
     }
 
-    public void setNumExecutors(int n) throws IOException {
+    public void setNumExecutors(@Nonnegative int n) throws IOException {
```

Comment:
No runtime effect AFAIK; should really be matched by an `IllegalArgumentException`.